docs/guide/advanced/sfc.md

```diff
@@ -311,6 +401,99 @@ You can use define locale messages for global scope with `global` attribute:
 
 In the above example, since the `global` attribute is set, the locale messages defined in `i18n` custom blocks can be merged as a resource for locale messages of global scope.
 
+### Define global scope
+
+If you are using one of `@intlify/vite-plugin-vue-i18n`, `@intlify/vue-i18n-loader`  or `@intlify/rollup-plugin-vue-i18n`
+plugin on your project, you can also define the `global` scope for all your `i18n` custom blocks
+on all your `SFC` using the `globalSFCScope` option.
+
+**Warning**: beware enabling `globalSFCScope: true`, all `i18n` custom blocks in all your `SFC` will be on `global` scope.
+
+For example, in order to configure `global` scope on all your `i18n` custom blocks on all your `SFC`:
+
+```html
+<!-- custom block equivalent to &lt;i18n lang="yml" global> -->
+<i18n lang="yml">
+en:
+  hello: Hello
+es:
+  hello: Hola
+</i18n>
+```
+
+or
+
+```html
+<!-- custom block equivalent to &lt;i18n src="./locales/myMessages.json" global> -->
+<i18n src="./locales/myMessages.json5">
+</i18n>
+```
+
+just configure `globalSFCScope: true` on your plugin configuration:
+
+#### Use global format with vite plugin
+
+```javascript
+import vue from '@vitejs/plugin-vue'
+import vueI18n from '@intlify/vite-plugin-vue-i18n'
+
+plugins: [
+  vue(),
+  vueI18n({
+    globalSFCScope: true,
+    /* other options */
+  })
+]
+```
+
+#### Use global format with webpack plugin
+
+```javascript
+module.exports = {
+  // ...
+  module: {
+    rules: [
+      {
+        test: /\.vue$/,
+        loader: 'vue-loader',
+      },
+      // for i18n resources (json/json5/yaml)
+      {
+        test: /\.(json5?|ya?ml)$/, // target json, json5, yaml and yml files
+        type: 'javascript/auto',
+        globalSFCScope: true,
```

Comment:
Same as the above my comment, In webpack, We don't still develop `defaultSSFCLang`.
Currently, we have a feature request.
https://github.com/intlify/bundle-tools/issues/24